charts/sysdig/templates/configmap-image-analyzer.yaml

```diff
@@ -46,17 +44,15 @@ data:
   no_proxy: {{ .Values.nodeImageAnalyzer.settings.noProxy }}
   {{- end -}}
 {{- end }}
-{{- if and (not .Values.nodeImageAnalyzer.deploy) .Values.nodeAnalyzer.deploy}}
+{{- if and (not (include "deploy-nia" .)) .Values.nodeAnalyzer.deploy}}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "sysdig.fullname" . }}-image-analyzer
   labels:
 {{ include "sysdig.labels" . | indent 4 }}
 data:
-  {{- if .Values.nodeAnalyzer.collectorEndpoint }}
-  collector_endpoint: {{ .Values.nodeAnalyzer.collectorEndpoint }}
-  {{- end }}
+  collector_endpoint: "https://{{ required "A valid .Values.nodeAnalyzer.apiEndpoint is required" (.Values.nodeAnalyzer.apiEndpoint | default .Values.nodeAnalyzer.collectorEndpoint) }}/internal/scanning/scanning-analysis-collector"
```

Comment:
Include the protocol prefix and the path, and also make this value mandatory

Reply:
We support both .Values.nodeAnalyzer.collectorEndpoint for backwards compatibility, but the value is not renamed to `apiEndpoint`to avoid confusion.